nms/packages/magmalte/app/components/Subscribers.js

```diff
@@ -42,6 +42,7 @@ import withAlert from '@fbcnms/ui/components/Alert/withAlert';
 import {Route} from 'react-router-dom';
 import {makeStyles} from '@material-ui/styles';
 import {map} from 'lodash';
+import {NETWORK_TYPES as networkTypes} from '../views/subscriber/SubscriberUtils';
```

Comment:
Don't rename the constant when you import it. The all caps naming convention implies it as a constant. Additionally, renaming this makes it harder to search up later.

Reply:
Got it.